keytypeoptions.md

```diff
@@ -0,0 +1,151 @@
+## KeyTypes Supported through -t verb
```

Comment:
Delete this file as well (I've added this to the exclusions in the script so it shouldn't get copied again).